lib/traject/numismatic_csv_config.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Numismatics CSV Mapping Configuration
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+  provide 'allow_empty_fields', true
+  provide 'agg_provider', 'Numismatics Society'
+end
+
+to_field 'agg_provider', provider
+
+# MET Museum
+to_field 'id', numismatic_id
+
+to_field 'cho_contributor', column('Authority')
```

Comment:
I'm wondering if the pipes in `Authority` indicate multiple values. ANS's site is down so I can't confirm.